### PR TITLE
New pull request FullProfAPP

### PR DESCRIPTION
**Pull request by oarcelus**
Includes a new link for direct download of the installer files.

GitLab repo link moved to `git_url`.

### CODE REVIEW

File: apps.yaml [hunk 1]
```diff
@@ -201,12 +201,14 @@ FullProfAPP:
...
         authors: Oier Arcelus, Juan Rodriguez-Carvajal, Nebil Ayape Katcho
-        external_url: https://code.ill.fr/rodriguez-carvajal/fullprof-app
+        external_url: https://www.ill.eu/sites/fullprof/php/downloads.html
         logo: https://code.ill.fr/rodriguez-carvajal/fullprof-app/-/raw/master/src/fpgui/icons/splash.png
         state: development
+        version: 1.0
     categories:
         - data-analysis
         - utilities
+    git_url: https://code.ill.fr/rodriguez-carvajal/fullprof-app
```
Comment (tsthakur) [inline apps.yaml:211]: this is not a github url, please remove it.

**Review by tsthakur — CHANGES_REQUESTED** [review]

File: apps.yaml (hunk 1 above)
Comment (oarcelus) [reply apps.yaml:211]: The URL links to a GitLab repo. This was the link that was included originally in the last pull request I did, I've just moved it from 'metadata' to 'git_url'.

File: apps.yaml [hunk 2]
```diff
@@ -201,12 +201,15 @@ FullProfAPP:
         description: |
             An PyQt5 based graphical user interface for the automated analysis of powder X-ray diffraction data based on the refinement engine FullProf.
         authors: Oier Arcelus, Juan Rodriguez-Carvajal, Nebil Ayape Katcho
-        external_url: https://code.ill.fr/rodriguez-carvajal/fullprof-app
-        logo: https://code.ill.fr/rodriguez-carvajal/fullprof-app/-/raw/master/src/fpgui/icons/splash.png
+        external_url: https://www.ill.eu/sites/fullprof/php/downloads.html
+        documentation_url: https://fullprofapp.readthedocs.io/en/stable/
+        logo: https://gitlab.com/d7081/fullprofapp/-/tree/main/src/fpgui/icons/splash.png
         state: development
+        version: 1.0
```
Comment (tsthakur) [inline apps.yaml:208]: This needs to be a string, so somethign like:
```suggestion
        version: '1.0'
```

**Review by tsthakur — CHANGES_REQUESTED** [review]

File: apps.yaml (hunk 1 above)
Comment (tsthakur) [reply apps.yaml]: This url is not working for me, so I cannot check if it is indeed a gitlab url. Please note that only github and gitalb url are allowed in the `git_url` field. If you want to link some other websites, you can use `external_url` field.

Comment (oarcelus) [reply apps.yaml]: These changes do not apply to the latest commit

**Pull request merged**